### ISSUE TITLE
Login is successful, but I drop back to the login screen

### ISSUE THREAD

**Issue by BSDGuyShawn** [SolDoesTech/HyprV2#14]
As mentioned in https://github.com/SolDoesTech/HyprV2/issues/13 I had an issue compiling waybar. I swapped out for waybar-hyprland-git as recommended and the rest of the install was uneventful.

When logging in the login completes successfully but I drop back to the login screen.

I am installing on a MacBook Pro - the first generation with the Retina display.

Any help is appreciated. Just let me know what you need from my end. I am new to Arch but am familiar with other Linux distros.

**Comment by MarceloNogueiraGit** [comment]
Hi, I'll try to be understandable with my comment, I'm did a installation of hyprv2 following the YouTube video, and I get stuck on login manager, I put the password and goes back to the login manager, my config it's a full amd computer, r5 5600x, rx 6600xt, 32gb ram ddr4 3200mhz, 1Tb lenovo sata 2.5 ssd, corsair ps cx650w, anyone knows how to fix that, or if my computer is not compatible?

**Comment by SolDoesTech** [comment]
Hello, please switch TTY with CTRL+ALT+F3 or F4 and login from a console then type Hyprland with a capital H. see if that launches Hyprland for you.

**Comment by BSDGuyShawn** [comment]
> Hello, please switch TTY with CTRL+ALT+F3 or F4 and login from a console then type Hyprland with a capital H. see if that launches Hyprland for you.

It does launch Hyprland but it is not really usable. display is funky. The shortcut keys do work and I have installed and used MS Edge as a browser and could use that, although screen placement is not correct and I can't move anything with my mouse.

**Comment by SolDoesTech** [comment]
Are you using a VM?

**Comment by BSDGuyShawn** [comment]
> Are you using a VM?

No sir. I am installing on bare metal. It is a MacBook Pro laptop. No dual boot or bootcamp or anything just a straight install like would be done on any other machine.

**Comment by SolDoesTech** [comment]
I noticed that in your original comment but had to confirm since this behavior is similar to what I see on a VM. If you do an ls -la in your home directory do you have a .start-hypr-nvidia or a .start- hypr-amd ? if its the -amd can you try executing it.

**Comment by BSDGuyShawn** [comment]
I am working at the moment but will take a look once I am off for the day.

**Comment by MarceloNogueiraGit** [comment]
> Hello, please switch TTY with CTRL+ALT+F3 or F4 and login from a console then type Hyprland with a capital H. see if that launches Hyprland for you.

That way doesn't work for me, console says that Hyprland not found

**Comment by SolDoesTech** [comment]
hmm yeah something went wrong for sure in the install, which explains why there re so many issues. please execute "sudo pacman -S hyprland" and  see if that installs it

**Comment by SolDoesTech** [comment]
Any chance I can get you guys to try out V3 instead of V2 and see if that works any better?
https://github.com/SolDoesTech/HyprV3

**Comment by BSDGuyShawn** [comment]
> Any chance I can get you guys to try out V3 instead of V2 and see if that works any better? https://github.com/SolDoesTech/HyprV3

I tried V3 over lunch. It dumps me to a black screen, and I can't get to any of the virtual terminals.
I did see a message that my card isn't supported and will be ignored so I rebooted to the install media, mounted / and /boot under /mnt and used arch-chroot to remove all nvidia drivers and installed the nouveau driver as outlined here:
https://wiki.archlinux.org/title/nouveau

I now get a desktop session, can switch between V1, V2 and V3.

I am sure my issues are a result of being on an older MacBook Pro - The first model with the Retina Display.
